data/hai/hai reveal 1 intro.txt

```diff
@@ -674,7 +676,7 @@ mission "Hai Leaks Response 2C"
 			label next
 			`	"Now, allow me to introduce you to Terry," Remington says while gesturing to the woman behind him.`
 			`	"Terry Adrianopoulos. Pleasure to meet you," she says with a broad smile and a slight bow. The young woman goes from calculating to gregarious in the blink of an eye as your attention shifts to her. She seems at once pleasant and approachable while at the same time being slightly off-putting and uncanny. It's an eerie feeling.`
-			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too please to hear that, but doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently," he says. You think that he may have glanced at Terry as he said that, but you aren't sure if it was just incidental or indicative of something. "Nothing I can discuss in detail, unfortunately, what with all the NDAs involved, but something I'll certainly enjoy taking a break from.`
+			`	"Terry will be accompanying me as... well, let's call her my secretary," Remington says with a sly smile. Terry seems none too pleased to hear that, but she doesn't correct him. "I have men who can hold the fort here while I'm gone, and I really do need to get some fresh air, so the two of us will be joining you. There is an internal political situation going on within the Syndicate that has been causing me quite the headache recently," he says. You think that he may have glanced at Terry as he said that, but you aren't sure if it was just incidental or indicative of something. "Nothing I can discuss in detail, unfortunately, but something I'll certainly enjoy taking a break from.`
```

Comment:
Remington is speaking here, or else this needs to be changed so we know that it's Terry who's speaking. I took it out because Remington is acting casual here, but I can add in something like "I'm not authorised to talk to you about it", or "the details will bore you, so I won't bother" to make it more clear.

Reply:
It is Remington speaking, but specifically calling out the NDA's is functionally just an example of corporate humour in this case. Most captains would probably have encountered an NDA if they've ever dealt with the Syndicate.
if the player doesn't get it.. well that's part of their character then.